src/_data/sidenav.yml

```
@@ -99,13 +99,8 @@
         - title: Flutter versus Swift concurrency
           permalink: /get-started/flutter-for/dart-swift-concurrency
 
-    - divider
-    - title: Codelabs
-      permalink: /codelabs
-    - title: Cookbook
-      permalink: /cookbook
-    - title: Demos and samples
-      permalink: https://github.com/flutter/samples
+- title: Guides and samples
```

I think I prefer "Tutorials and samples" as "guides" has been used elsewhere and it better matches the "types" the page covers:

```suggestion
- title: Tutorials and samples
```